### ISSUE TITLE
Allow equals sign to be omitted for named arguments

### ISSUE THREAD

**Issue by dantownsend** [piccolo-orm/targ#16]
You currently have to use an equals sign when passing in a named argument:

```
python main.py my_command --desc="some message"
```

This can lead to confusion when people omit it, using this format instead:

```
python main.py my_command --desc "some message"
```

It would be good to support both formats.

Originally reported here: https://github.com/piccolo-orm/piccolo/issues/418

**Comment by akankshasainics** [comment]
Hello!  Can I work on this ?

**Comment by dantownsend** [comment]
@akankshasainics Yes, that would be good. It might be quite a challenging issue though!

**Comment by akankshasainics** [comment]
Thanks, I will try.
